config/locales/en.yml

```diff
@@ -78,6 +78,18 @@ en:
     claims_preventing_payment_message:
       one: This claim cannot currently be approved because we’re already paying another claim (%{claim_references}) to this claimant in this payroll month using different payment details. Please speak to a Grade 7.
       other: This claim cannot currently be approved because we’re already paying other claims (%{claim_references}) to this claimant in this payroll month using different payment details. Please speak to a Grade 7.
+    payment_deletion_claim_warning:
+      one: This means claim %{references} has not been included in this pay run.
+      other: This means claims %{references} have not been included in this pay run.
```

Comment:
```suggestion
      other: This means claims %{references} will not be included in this pay run.
```

Reply:
Yeah, I talked to Faye about this, I think her thinking was that we are deleting this because it hasn't been included in the confirmation report, but I agree, it is a bit confusing.